src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -1071,6 +1071,22 @@ static inline int rom_get_last_boot_type(void) {
  */
 int rom_add_flash_runtime_partition(uint32_t start_offset, uint32_t size, uint32_t permissions);
 
+/*! \brief  Pick A/B partition for a separate partition
+ * \ingroup pico_bootrom
+ *
+ * This will perform extra checks to prevent disrupting a main image TBYB, and return errors
+ * 
+ * Also checks that the chosen partition contained a valid image
+ *
+ * \param workarea_base base address of work area
+ * \param workarea_size size of work area
```

Comment:
Presumably there's some kind of minimum size that the workarea needs to have?

Reply:
It's documented as 3064 in the datasheet and other doxygen (this is the workarea for the `rom_pick_ab_partition_function`) and recommends the use of 3k (3072), but I believe that's too small - I always just use 4k (4096) which works

Reply:
Just in case I was being too subtle - I was trying to ask if this minimum size should be mentioned in _this_ doxygen?

Reply:
Yeah, I didn't want to add an incorrect size to my new definition - double checking with @kilograham but I think the correct size for the workarea (for `rom_pick_ab_partition` and `rom_get_uf2_target_partition`) is 3264, obtained by calling `rom_pick_ab_partition` with increasing size until it returns a non-negative value. So it would probably be best to recommend a size of 3.25k (0xD00, 3328)?

Reply:
I've added a note that it requires the same minimum workarea size as `rom_pick_ab_partition` - the doxygen for that and `rom_get_uf2_target_partition` should still be updated to say the correct minimum workarea size, but I'll pop that in a separate PR